### PR TITLE
ORV2-1852 - Update refund calculation for void

### PR DESCRIPTION
**Pull request by zgong-gov**
# Description

Please provide a summary of the change and the issue fixed. Please include relevant context. List dependency changes.

- Update refund calculation for void (always be net value rather than 30-day increments)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Create a new permit with any duration. Go to the staff search permits page to void this permit, and confirm that the amount refunded is indeed the same as the duration.
- [x] Create a new permit with any duration, and amend (with refund) it once or more. Then proceed to void this permit and observe that the amount refunded is equal to the net amount.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1090-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1090-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1090-dops.apps.silver.devops.gov.bc.ca/api)
- [TPS-Migration](https://onroutebc-1090-tps-migration.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/features/permits/helpers/feeSummary.ts [hunk 1]
```diff
@@ -111,14 +111,11 @@ export const calculateAmountForVoid = (
 ) => {
   const permitState = getPermitState(permit);
   if (
-    permitState === PERMIT_STATES.EXPIRES_IN_30 ||
     permitState === PERMIT_STATES.EXPIRED
   ) {
     return 0;
   }
 
   const netPaid = calculateNetAmount(permitHistory);
```
Comment (erikataot) [inline frontend/src/features/permits/helpers/feeSummary.ts]:
```suggestion
  return calculateNetAmount(permitHistory);
```

File: frontend/src/features/permits/helpers/feeSummary.ts [hunk 2]
```diff
@@ -111,14 +111,11 @@ export const calculateAmountForVoid = (
...
-  const daysBeforeExpiry = daysLeftBeforeExpiry(permit);
-  const incrementsOf30DaysLeft = Math.floor(daysBeforeExpiry / 30);
-  return Math.min(incrementsOf30DaysLeft * 30, netPaid);
+  return netPaid;
```
Comment (erikataot) [inline frontend/src/features/permits/helpers/feeSummary.ts]:
```suggestion

```

**Review by erikataot — APPROVED** [review]
made a suggestion - but approved

File: frontend/src/features/permits/helpers/feeSummary.ts (hunk 1 above)
Comment (zgong-gov) [reply frontend/src/features/permits/helpers/feeSummary.ts]: Ok sure I'll change this

File: frontend/src/features/permits/helpers/feeSummary.ts (hunk 2 above)
Comment (zgong-gov) [reply frontend/src/features/permits/helpers/feeSummary.ts]: ok

**Review by krishnan-aot — APPROVED** [review]

**Pull request merged**